tools/cockpit.spec

```diff
@@ -454,11 +454,16 @@ Requires: glib2 >= 2.50.0
 Conflicts: firewalld < 0.6.0-1
 Recommends: sscg >= 2.3
 Recommends: system-logos
-Requires: systemd >= 235
 Suggests: sssd-dbus
+%if !0%{?suse_version}
+Requires: systemd >= 235
 Requires(post): systemd
 Requires(preun): systemd
 Requires(postun): systemd
+%else
+Requires: group(wheel)
+Requires(pre): permissions
```

Comment:
What does that do? Can you please give a rationale to the commit message?

Cockpit doesn't really care about the `wheel` group. I just see that we still reference it here, though:

    %attr(775, -, wheel) %{_localstatedir}/lib/cockpit

We shouldn't create that directory at all any more, I'll send a PR to eliminate it. If  it was just about that, then please drop this bit.

Reply:
I sent PR #14588 to clean up that bit.

Reply:
I guess it's about that so will remove